pkg/views/user/create/view.go

```diff
@@ -54,9 +55,13 @@ func CreateUserView(createView *CreateView) {
 				Validate(func(str string) error {
 					if str == "" {
 						return errors.New("password cannot be empty")
-					}
+					} else if !IsValidPassword(str)  {
+					 	return errors.New("password is incorrect")
```

Comment:
improve the error message on user create.

`Password is incorrect` is misleading.

`Password should contain 8-128 characters long with at least 1 uppercase, 1 lowercase and 1 number.` would be a good error message or